app/src/main/java/org/thunderdog/challegram/navigation/DrawerItemView.java

```diff
@@ -187,6 +194,16 @@ public void setAvatar (Tdlib tdlib, long chatId) {
     this.receiver.requestChat(tdlib, chatId, AvatarReceiver.Options.NONE);
   }
 
+  public void setEmojiStatus (TdlibAccount account) {
+    emojiStatusHelper.updateEmoji(account.tdlib(), account.getUser(), new TextColorSetOverride(TextColorSets.Regular.NORMAL) {
```

Comment:
`account.tdlib()` call here "wakes up" `Tdlib` instance. This means that if user has `10` logged in accounts that are visible in the accounts list, it will in parallel launch all of them (without explicit need), which will cause app crashes on low-end devices / high amount of accounts.

Reply:
In order to have some of the account properties needed without launching TDLib instance [TdlibAccount.DisplayInformation](https://github.com/TGX-Android/Telegram-X/blob/23d838cb7c41fe4e6883dc7256e349cc447ac5cf/app/src/main/java/org/thunderdog/challegram/telegram/TdlibAccount.java#L471) is used (refer to e.g. `profilePhotoSmallPath` - path to fully downloaded profile picture).

Reply:
Here's an example how avatar placeholder is being restored without launching `Tdlib` instance: [TdlibAccount.java:726](https://github.com/TGX-Android/Telegram-X/blob/23d838cb7c41fe4e6883dc7256e349cc447ac5cf/app/src/main/java/org/thunderdog/challegram/telegram/TdlibAccount.java#L726)

`DisplayInformation` is being refreshed after every TDLib launch caused by other events in the app: incoming Firebase message, [application launch (with limit in amount of parallel Tdlib instances)](
https://github.com/TGX-Android/Telegram-X/blob/23d838cb7c41fe4e6883dc7256e349cc447ac5cf/app/src/main/java/org/thunderdog/challegram/MainActivity.java#L149), manual selection.